Improve Documentation (#315)
dev-sec/ansible-os-hardening: README.md

      net.ipv4.ip_forward: 1
```

### sysctl - vm.mmap_rnd_bits

> We are setting this sysctl to a default of `32`, some systems only support smaller values and this will generate an error. Unfortunately we cannot determine the correct aplicable maximum. If you encounter this error you have to override this sysctl in your playbook.

Reviewer: ```suggestion
We are setting this sysctl to a default of `32`, some systems only support smaller values and this will generate an error. Unfortunately we cannot determine the correct applicable maximum. If you encounter this error you have to override this sysctl in your playbook.
```

```yaml
- hosts: localhost
  roles:
    - dev-sec.os-hardening
  vars:
    sysctl_overwrite:
      vm.mmap_rnd_bits: 16
```

> We know, this is the case on Paspberry Pi and possibly some other ARM based devices.

Reviewer: ```suggestion
We know that this is the case on Raspberry Pi.
```
